client/reader/following-manage/empty.jsx

```diff
@@ -36,12 +36,12 @@ class FollowingManageEmptyContent extends React.Component {
 
 		return (
 			<EmptyContent
-				className="is-drake"
+				className="is-following-manage-empty"
```

Comment:
I originally used `is-drake` as a modifier to apply different spacing all of the empty pages that used Drake.

I think we can just drop the `is-` prefix now and use `following-manage__empty` here.